### PR TITLE
GitHub Actions: Switch to blocklist approach for  clazy checks

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

File: .github/workflows/clazy.yml
Comment (Holzhaus) [inline .github/workflows/clazy.yml:25]: After #3302 has been merged, we can remove the last two:

```suggestion
        CLAZY_CHECKS: level1,no-rule-of-two-soft,no-non-pod-global-static,no-qproperty-without-notify,no-wrong-qevent-cast
```

**Comment by Holzhaus** [comment]
Clazy check passed, shall we merge? I don't think we need to wait for CI for this trivial change.

**Review by uklotzde — APPROVED** [review]
LGTM

**Pull request merged**
